api/python/index.md

```diff
@@ -76,6 +75,14 @@ to the Python Client API package.
     This route is particularly useful if you are building applications that
     depend on this API, and are deploying them to hosting services.
 
-You are now ready to go! From here you can view our tutorials on how to use the
+    ```python
```

Comment:
This looks repeated from the first 1. under "You can set the API key by calling `datacommons.set_api_key`" ?